src/materials/MeshDepthMaterial.js

```diff
@@ -33,6 +33,10 @@ THREE.MeshDepthMaterial = function ( parameters ) {
 	this.wireframe = false;
 	this.wireframeLinewidth = 1;
 
+	// far clipping plane in both RGBA and Basic encoding
+	this.clearColor = new THREE.Color( 1.0, 1.0, 1.0 );
+	this.clearAlpha = 1.0;
```

Comment:
They are automatically used by EffectRenderer if you do not specify overrides when using renderPass, renderOverride:

https://github.com/mrdoob/three.js/pull/8626/files/53b5fb0548bd0a82494d3bd7f2e68ec24b15b1f7#diff-2bb4aa115364b94510e8f396157285c7R37

Most people do not know that you have to clear depth so that it defaults to a far distance where polygons are not written.  The same with the normal material's clear color/clearAlpha: 

https://github.com/mrdoob/three.js/pull/8626/files/53b5fb0548bd0a82494d3bd7f2e68ec24b15b1f7#diff-717b15a206ca91fed8cb9df26d53be71R23

We do not need to have them in the materials, but it is really useful to have these in the materials themselves as they are really required for that material to work properly.  If you clear depth with black you are saying everything is close, which is generally wrong.  If you clear normal with black or white, you are clearing it with what are invalid normals, which will cause issues when you try to make sense of them in a post effect.